mentees/README.md

```diff
@@ -45,29 +46,30 @@ Mentorship programs can be highly rewarding, yet challenging commitments to unde
 
 In summary, there are multiple contacts and resources available to guide you, and a supportive network can play a crucial role in the success and fulfillment of your mentorship experience.
 
-## **Programs**
+## Programs
 
 There are numerous open source mentoring and internship programs available, with each emphasising different criteria such as specific skillsets, demographics, industries and specialisations. 
 
 Currently, the CNCF concentrates on a limited selection of such programs; featuring a range of projects to encourage interest from a broad pool of talent and backgrounds. 
 These are as follows:
 
-**LFX**
+**LFX**  
 The [Linux Foundation Mentorship program](https://docs.linuxfoundation.org/lfx/mentorship/mentee-guide/introduction), also known as '**LFX**', plays a crucial role in providing mentees, including students and both new and experienced programmers; with structured hands-on learning opportunities in open source software development. 
 Through the program, mentees are paired with mentors based on their skills and interests to gain valuable experience by participating in and contributing to open source projects, all while [getting paid for it](https://docs.linuxfoundation.org/lfx/mentorship/mentees).
  
-**Google Summer of Code**
+**Google Summer of Code**  
 The [Google Summer of Code](https://summerofcode.withgoogle.com/), often abbreviated to GSoC, is an international annual program in which Google awards stipends to contributors who successfully complete a free and open-source software coding project during the summer. 
 
-**Google Season of Docs**
+**Google Season of Docs**  
 [Google Season of Docs](https://developers.google.com/season-of-docs/docs/organization-application-hints) is the ideal opportunity for aspiring and professional technical writers interested in non-code contribution, providing support for open source projects to improve their documentation. 
 
 **Outreachy**
```

Comment:
This is missing the extra two spaces to make a linebreak

Reply:
Good catch!